test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -202,6 +202,16 @@ export default class BaseFormObject {
       .child('label');
   }
 
+  getFormFieldInputOptional(fieldName) {
```

Comment:
nit: Can we rename this `getFormFieldSubLabel` so it's a bit more clear what we're looking for?